packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -17,8 +22,38 @@ const plugin = async (): Promise<PluginOption[]> => {
         process: "build",
       },
     }),
+    cleanup(projectStructure),
   ];
 };
 
+const clientHydration = async (
+  projectStructure: ProjectStructure
+): Promise<Plugin> => {
+  return {
+    name: "client-hydration:build",
+    apply: "build",
+    buildStart: {
+      sequential: true,
```

Comment:
Is this needed or order necessary?

Reply:
Yup necessary because buildStart is normally parallel